themes/command_duration.theme.bash

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+if [ -z "$BASH_IT_COMMAND_DURATION" ] || [ "$BASH_IT_COMMAND_DURATION" != true ]; then
+    return
+fi
+
+# Define tmp dir and file
+COMMAND_DURATION_TMPDIR="${TMPDIR:-/tmp}"
+COMMAND_DURATION_FILE="$COMMAND_DURATION_TMPDIR/bashit_theme_execution_$BASHPID"
+
+COMMAND_DURATION_ICON='  '
```

Comment:
change it to only set when the variable is not already set

Reply:
Fixed by 790252cc9b1a520fb81db485bf3ccabfcddbaaef